fe1-web/src/resources/strings.ts

```diff
@@ -397,15 +397,23 @@ namespace STRINGS {
     ', you can continue and start using the POPStellar application. ' +
     'The purpose of this application is to demonstrate use cases for proofs of personhood (PoP).';
 
-  export const wallet_welcome_start_exploring = 'Start exploring POPStellar 🤩';
+  export const wallet_welcome_already_know_seed_1 =
+    'If you have already used this application before and remember your previous';
+  export const wallet_welcome_already_know_seed_2 =
+    ', you can restore your wallet by pressing the respective button.';
```

Comment:
does it make sense that we start with a comma ?

Reply:
This sentence is split into two parts with "seed 🔑" in between (colored in blue). If we would have it the same color or put the comma inside a <Text/> as with the spaces, we would could combine the two / not start the sentence with a comma. What do you think?

<img width="562" alt="Screenshot 2022-12-20 at 14 42 21" src="https://user-images.githubusercontent.com/2634739/208927529-7bfa8e2d-efa8-45df-98ee-6b2d6d0e3f52.png">